clang-tools-extra/docs/clang-tidy/checks/modernize/use-using.rst

```diff
@@ -28,6 +28,15 @@ After:
   using R_t = struct { int a; };
   using R_p = R_t*;
 
+The checker ignores `typedef` within `extern "C" { ... }` blocks.
```

Comment:
Maybe this should be put into configuration option ? To give an option to preserve current behavior.

Reply:
Would it be preferred to make this an option ?

Reply:
I think so, for example I would like to enable this for my projects.
Ignoring extern "C" makes only sense if same header is compile in C and C++ mode.
If header is compiled only in C++ mode, but uses extern "C" simply to properly handle symbols mangling, then there is no need to ignore such typedefs.